github/plugin.py

```diff
@@ -82,8 +82,23 @@ def __init__(self):
                 "user_data_required": True,
                 "help": "Delete a GitHub repository.",
                 "enabled": True     # It is HIGHLY recommended you have auth enabled for this!!
+            },
+            "!InviteUser": {
+                "command": "!InviteUser",
+                "func": self.add_user_to_org_command,
+                "user_data_required": True,
+                "help": "Adds a GitHub user to a specific GitHub organization.",
+                "permitted_permissions": ["admin", "member"],
+                "enabled": True
+            },
+            "!AddUserToTeam": {
+                "command": "!AddUserToTeam",
+                "func": self.add_user_to_team_command,
+                "user_data_required": True,
+                "help": "Adds a GitHub user to a specific team inside the organization.",
+                "permitted_permissions": ["maintainer", "member"],
```

Comment:
Change name to `permitted_roles` to be consistent with GitHub verbiage.  Also, `member` should be the default option and perhaps the role made to be optional.  We would have to test if optional parameters can be provided.